pkg/kubelet/apis/config/types.go

```diff
@@ -407,6 +421,17 @@ type KubeletConfiguration struct {
 	// class value that lies in the range of that value and the next higher entry in the
 	// list when the node is shutting down.
 	ShutdownGracePeriodByPodPriority []ShutdownGracePeriodByPodPriority
+	// GracefulNodeShutdownPodPolicy defines policy for how pods are handled during graceful shutdown.
+	// The policy types available are `SetTerminal` (default) or `LeaveRunning`.
+	// `SetTerminal` will result in pods being to set terminal phase during shutdown (enabled by default).
+	// If a node becomes ready again after graceful node shutdown (e.g. a reboot), will result in pods not being started on the same node
+	// since the pods will be placed into terminal phase.
+	// In contrast, with `LeaveRunning` policy, pods will not be placed into terminal phase,
+	// and as result they will resume to run after the node becomes ready again.
+	// Default: true
```

Comment:
Please update the default here

```suggestion
	// Default: SetTerminal
```

Reply:
Thanks, updated.